tests/test_sphinx_multitoc_numbering.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
+
+import pytest
+from bs4 import BeautifulSoup
+
+# from jupyter_book.cli import main as commands
+
+
+@pytest.mark.parametrize(
+    "toc_file",
+    (
+        "_toc_numbered.yml",  # Numbered at top-level
+        "_toc_numbered_depth.yml",  # Numbered at top-level, limited to depth 1
+        "_toc_numbered_parts.yml",  # Numbered at top-level w/ parts
+        "_toc_numbered_parts_subset.yml",  # Only some sections numbered
+        "_toc_numbered_depth_parts_subset.yml",  # Selected numbering limited to depth 1
+    ),
+)
+def test_toc_numbered_multitoc_numbering_false(
+    toc_file, cli, build_resources, file_regression
+):
+    """Testing use_multitoc_numbering: false"""
+    books, tocs = build_resources
+    config = books.joinpath("config").joinpath("_config_sphinx_multitoc_numbering.yml")
+    toc = tocs.joinpath(toc_file)
+    # result = cli.invoke(
```

Comment:
is there a reason that all of this is commented out?

Reply:
@choldgraf It was commented out by @mmcky in favor of the code below owing to this issue https://github.com/executablebooks/jupyter-book/issues/1339

Reply:
@choldgraf I spent ages trying to fix this `bug` but in the end I used a subprocess to ensure an independent run. As @AakashGfude indicates I opened a tracking issue and added a comment in the code. 

https://github.com/executablebooks/jupyter-book/issues/1339

Reply:
@mmcky ahh gotcha - that makes sense. Is there a reason that the explanatory comment goes below instead of above this commented out block? I think that's what was confusing. Maybe we can apply the suggestion above to make it clearer?

Reply:
ah just saw this @choldgraf I will move it.